src/common.ts

```diff
@@ -311,22 +311,28 @@ export function civ() {
 
 	// Based on the input, set the values for the civData object.
 	switch (input) {
-		case 'GHub':
-			civData.galaxy = 'Euclid';
-			civData.civilized = 'Galactic Hub Project';
-			civData.civStub = 'GHub';
+		case 'ZikCORE':
+			civData.galaxy = 'Zikehpm';
+			civData.civilized = 'Zikehpm CORE';
+			civData.civStub = 'ZikCORE';
```

Comment:
New civs are not eligible for a banner, therefore the `civStub` property can be removed.
https://nomanssky.fandom.com/wiki/No_Mans_Sky_Wiki:Explained_-_Civilized_Space_Banners_and_Usage